### PR TITLE
docs: Document how to ignore Express routes

### PR DESCRIPTION
**Pull request by MadLittleMods**
## Which problem is this PR solving?

Fix https://github.com/open-telemetry/opentelemetry-js-contrib/issues/1034

## Short description of the changes

Document how to ignore Express routes

## Checklist

- [ ] ~~Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.~~

### CODE REVIEW

**Review by vmarchaud — APPROVED** [review]

**Comment by vmarchaud** [comment]
@MadLittleMods I believe you can ignore unit tests failure which isn't linked to your change but the markdown isn't correctly formated, could you fix it please ?

File: plugins/node/opentelemetry-instrumentation-express/README.md [hunk 1]
```diff
@@ -78,6 +78,31 @@ Express instrumentation has few options available to choose from. You can set th
...
+      ignoreIncomingRequestHook(req) {
+        // Ignore spans from static assets.
+        const isStaticAsset = !!req.url.match(/^\/static\/.*$/);
+        return isStaticAsset;
+      }
+    }),
+    new ExpressInstrumentation(),
+  ],
+});
+```
```
Comment (MadLittleMods) [inline plugins/node/opentelemetry-instrumentation-express/README.md:103]: > the markdown isn't correctly formated, could you fix it please ?
>
> *-- https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1095#issuecomment-1191244963*

I'm not sure what you mean? Perhaps the auto-wrap at 80 characters? Is there a different preference?

For reference, it looks fine in the rendered view, https://github.com/open-telemetry/opentelemetry-js-contrib/blob/23ccd953f402aa1dd349c4304c64f25b55b8da18/plugins/node/opentelemetry-instrumentation-express/README.md

Comment (MadLittleMods) [reply plugins/node/opentelemetry-instrumentation-express/README.md]: @vmarchaud Friendly ping for another round of review to clarify this point ^

Comment (vmarchaud) [reply plugins/node/opentelemetry-instrumentation-express/README.md]: Totally sorry for not seeing this, checkout https://github.com/open-telemetry/opentelemetry-js-contrib/runs/8022040414?check_suite_focus=true

Comment (MadLittleMods) [reply plugins/node/opentelemetry-instrumentation-express/README.md]: Ahh, the CI linting. Thank you!

Fixed up ✅

**Comment by vmarchaud** [comment]
friendly ping @open-telemetry/javascript-approvers

**Comment by MadLittleMods** [comment]
Thanks for the review and merge @vmarchaud 🙂

**Pull request merged**